oxd-server/src/main/java/org/gluu/oxd/server/OxdServerApplication.java

```diff
@@ -16,7 +16,12 @@
 
     public static void main(String[] args) {
         try {
-            new OxdServerApplication().run(args);
+            String mode = args[0];
+            if ("stop".equals(mode)) {
+                ServerLauncher.shutdown(true);
+            } else {
+                new OxdServerApplication().run(args);
+            }
         } catch (Throwable e) {
             LOG.error("Failed to start oxd-server.", e);
```

Comment:
if we trigger stop and it fails, log message will say that it failed to start which is not accurate.

Reply:
Added below conditional statement in catch block to identify "start" or "stop" of server.